spec.html

```diff
@@ -31055,14 +31055,23 @@ <h1>get RegExp.prototype.dotAll</h1>
         <p>`RegExp.prototype.dotAll` is an accessor property whose set accessor function is *undefined*. Its get accessor function performs the following steps:</p>
         <emu-alg>
           1. Let _R_ be the *this* value.
-          1. If Type(_R_) is not Object, throw a *TypeError* exception.
-          1. If _R_ does not have an [[OriginalFlags]] internal slot, then
-            1. If SameValue(_R_, %RegExp.prototype%) is *true*, return *undefined*.
-            1. Otherwise, throw a *TypeError* exception.
-          1. Let _flags_ be _R_.[[OriginalFlags]].
-          1. If _flags_ contains the code unit 0x0073 (LATIN SMALL LETTER S), return *true*.
-          1. Return *false*.
+          1. Let _cu_ be the code unit 0x0073 (LATIN SMALL LETTER S).
+          1. Return ? RegExpHasFlag(_R_, _cu_).
         </emu-alg>
+
+        <emu-clause id="sec-regexphasflag" aoid="RegExpHasFlag">
+          <h1>RegExpHasFlag ( _R_, _cu_ )</h1>
```

Comment:
Nit: preference for `_codeUnit_` over `_cu_`.

Reply:
Done.